Bottom: 1010b23199c55904432edf0d944b316ad242ce3a
Top:    ef9c1d416a99aa9eb3aea2c3c497e96bb16f0712
Author: Ian Kent <raven@themaw.net>
Date:   2013-02-27 12:44:34 +0800

Refresh of autofs-5.0.7-add-symlink-pseudo-option.patch

---

diff --git a/modules/mount_autofs.c b/modules/mount_autofs.c
index 6f2e096..8c1e600 100644
--- a/modules/mount_autofs.c
+++ b/modules/mount_autofs.c
@@ -51,7 +51,7 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name,
 	int argc, status;
 	int nobind = ap->flags & MOUNT_FLAG_NOBIND;
 	int ghost = ap->flags & MOUNT_FLAG_GHOST;
-	int symlink = ap->flags & MOUNT_FLAG_SYMLINK;
+	int symlnk = ap->flags & MOUNT_FLAG_SYMLINK;
 	time_t timeout = ap->entry->maps->exp_timeout;
 	unsigned logopt = ap->logopt;
 	struct map_type_info *info;
@@ -122,7 +122,7 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name,
 			else if (strncmp(cp, "browse", 6) == 0)
 				ghost = 1;
 			else if (strncmp(cp, "symlink", 7) == 0)
-				symlink = 1;
+				symlnk = 1;
 			else if (strncmp(cp, "timeout=", 8) == 0) {
 				char *val = strchr(cp, '=');
 				unsigned tout;
@@ -161,7 +161,7 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name,
 	}
 	nap = entry->ap;
 	nap->parent = ap;
-	if (symlink)
+	if (symlnk)
 		nap->flags |= MOUNT_FLAG_SYMLINK;
 
 	argc = 1;
